BlackJackGame/Game.cs

```diff
@@ -26,53 +26,60 @@ namespace Black_Jack_Game
     public class Game
     {
         IConsole _newConsole;
+        
         static void Main(string[] args)
         {
             Deck deck = new Deck();
+
             List<string> orderedDeck = deck.GenerateDeck();
             List<string> shuffledDeck = deck.Shuffle(orderedDeck);
 
             Game game = new Game(new ConsoleActions());
+
+            List<string> playersHand = game.DealFirstDrawCard(deck, shuffledDeck);
             
-            
-            List<string> playersHand = new List<string>();
-            for (int i = 0; i < 2; i++)
-            {
-                string card = deck.DrawCard(shuffledDeck);
-                playersHand.Add(card);
-            }
+            List<string> dealersHand = game.DealFirstDrawCard(deck, shuffledDeck);
+
             game.PlayersTurn(playersHand, deck, shuffledDeck);
             
-            
-            List<string> dealersHand = new List<string>();
-            for (int i = 0; i < 2; i++)
-            {
-                string dealerCard = deck.DrawCard(shuffledDeck);
-                dealersHand.Add(dealerCard);
-            }
             //game.DealersTurn(playersHand, deck, shuffledDeck);
-            
-            Console.WriteLine(shuffledDeck.Count);
+
         }
         
         
-
         public Game(IConsole console)
         {
             _newConsole = console;
         }
 
-        public List<string> PlayersTurn(List<string> playersHand, Deck deck, List<string> shuffledDeck)
+
+        public List<string> DealFirstDrawCard(IDeck deck, List<string> shuffledDeck)
         {
-            string _playersOption = "1";
-            while (_playersOption == "1")
+            List<string> usersHand = new List<string>();
+            
+            for (int i = 0; i < 2; i++)
```

Comment:
This is just personal preference but if the number of loops is small and not likely to change (as in your case where this loop will always loops twice) I would personally not bother with it and do it like
```
usersHand.Add(deck.DrawCard(shuffledDeck))
usersHand.Add(deck.DrawCard(shuffledDeck))
```